src/Components/Wizard/Wizard.tsx

```diff
@@ -125,20 +125,20 @@ export class Wizard extends React.Component<WizardProps, WizardState> {
     if (this.isLastStep) {
       onComplete && onComplete(values, actions)
     } else {
-      actions && actions.setSubmitting(false)
+      actions?.setSubmitting(false)
       // If exist, call onSubmit on the current step
       const onSubmit = this.currentStep.props.onSubmit
       if (onSubmit) {
-        actions.setSubmitting(true)
+        actions?.setSubmitting(true)
```

Comment:
Ran into the following error while executing the spec without these undefined guards. Not sure how to mock this formik behavior.

```
    TypeError: Cannot read property 'setSubmitting' of undefined

      130 |       const onSubmit = this.currentStep.props.onSubmit
      131 |       if (onSubmit) {
    > 132 |         actions.setSubmitting(true)
          |                 ^
      133 |         const result = onSubmit(values, actions)
      134 | 
      135 |         if (result instanceof Boolean) {

      at setSubmitting (src/Components/Wizard/Wizard.tsx:132:17)
      at next (src/Components/Authentication/Mobile/LoginForm.tsx:195:13)
```

Reply:
A safe addition in any case 👍